### PR TITLE
React toggle switch component

### PR DESCRIPTION
**Pull request by khan-osama**
https://user-images.githubusercontent.com/63980419/112742681-5bc3b500-8f45-11eb-967f-233d07f69c19.mp4

### CODE REVIEW

File: react-toggle-switch-component/dist/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+input:checked + .circle {
+  background-color: rgb(2, 173, 2);
+}
+
+input:checked + .circle::before {
+  -webkit-transform: translateX(25px);
+  -ms-transform: translateX(25px);
+  transform: translateX(25px);
+}
```
Comment (thebearingedge) [inline react-toggle-switch-component/dist/styles.css:64]: If you can explain all of this CSS rule to me, I will accept your PR. For real.
- what is `:checked`
- what is `+`
- what is `::before`
- what is `-webkit-transform`, `-ms-transform`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: react-toggle-switch-component/dist/styles.css (hunk 1 above)
Comment (khan-osama) [reply react-toggle-switch-component/dist/styles.css:64]: :checked is a pseudo class that checks if a button is checked. ::before let's you insert content I believe within the html element in relation to the actual content of that element. I know conversely there is also an ::after which does something similar. With both of those then you can use the '+' to select sibling elements. So I wanted to have the circle move once the input box was checked. Webkit and ms transform....honestly I don't know. I've used it before in a lot of the previous assignments as I've seen it come up multiple times.

**Review by thebearingedge — APPROVED** [review]

File: react-toggle-switch-component/dist/styles.css (hunk 1 above)
Comment (thebearingedge) [reply react-toggle-switch-component/dist/styles.css:64]: It's a "vendor prefix".
https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix

Comment (thebearingedge) [reply react-toggle-switch-component/dist/styles.css:64]: Approved!

**Pull request merged**